### ISSUE TITLE
Keyboard Shortcuts for Actions Menu Button Examples

### ISSUE THREAD

**Issue by jscholes** [w3c/aria-practices#1688]
In the two "Actions Menu Button" examples (using [`element.focus()`](https://w3c.github.io/aria-practices/examples/menu-button/menu-button-actions.html) and [`aria-activedescendant`](https://w3c.github.io/aria-practices/examples/menu-button/menu-button-actions-active-descendant.html)), printable characters can be typed to move through the menu items.  However:

* The mark-up for the `menuitem`s doesn't include [aria-keyshortcuts](https://w3c.github.io/aria/#aria-keyshortcuts), so screen readers are unable to surface the presence of mnemonics.
* The mnemonic for each item is the same (the letter "A"), because each item begins with the same substring of "Action" followed by a space.  It would perhaps be better practice to indicate that menu items should have unique mnemonics where possible, allowing "1", "2", "3" or "4" to be pressed to activate the corresponding action directly.  Even though this is a toy example, the character-based navigation is useless because it doesn't save keystrokes.

**Comment by jscholes** [comment]
An additional note to add to this ticket: in the code for these examples, Space is explicitly bound to menu item activation:

```
switch (key) {
case ' ':
case 'Enter':
  this.closePopup();
  this.performMenuAction(tgt);
  this.buttonNode.focus();
  flag = true;
  break;
```

However, two things to consider:

1. The "Keyboard Support" -> "Menu" sections of the examples don't mention this fact; only Enter is listed as the activation key.
2. Space is a non-standard method of activating a menu item, at least on Windows.  On macOS it seems to work without needing to have Quick Nav enabled, but I'm not sure if it's something expected and actively used.

One way or another, the inconsistency between the code and documentation should be resolved.  Either by removing Space as a supported keystroke for item activation, or by listing it in the table of supported keyboard affordances.
